orchestrator/assets/business_travel.py

```diff
@@ -89,8 +90,17 @@ def travel_spending(
         table_exists = result.scalar()
         last_update = datetime(2020, 1, 1, tzinfo=pytz.UTC)
         if table_exists:
-            result = conn.execute(text(f"SELECT last_update FROM raw.{target_table}"))
-            last_update = pytz.utc.localize(result.scalar())
+            date_query = f"""
+                            SELECT trip_end_date, last_update
```

Comment:
Each quarter, we got a travel reimbursement sheet from VPF. with a trip_end_date (originally `Sent for Payment Date`) for each entry. The query string here is to get the latest entry to get both trip_end_date, and last_update (when this entry was last updated). 

Originally the logic used only `last_update`, so when listing out objects from a S3 bucket, only files updated after `last_update` will got ingested. It worked fine if we only ingest complete Quarter. But for FY24 Q2, when I developed this pipeline, I got a partial FY24Q2 till November 30. To avoid duplicate when appending new entries, I aded `trip_end_date` to filter out the duplicates.